autoload/nvimdev.vim

```diff
@@ -145,9 +147,6 @@ function! nvimdev#init(path) abort
   augroup nvimdev
     autocmd!
     autocmd BufRead,BufNewFile *.h set filetype=c
-    if get(g:, 'nvimdev_auto_lint', 1)
```

Comment:
indeed I missed that but what if I want to completely disable it ? even with my init.vim autocmd ? It's not possible in current state.